src/Microsoft.DotNet.Interactive/Commands/SendValue.cs

```diff
@@ -14,6 +15,11 @@ public class SendValue : KernelCommand
         FormattedValue formattedValue = null,
         string targetKernelName = null) : base(targetKernelName)
     {
+        if (formattedValue is null)
+        {
+            formattedValue = FormattedValue.CreateSingleFromObject(value, JsonFormatter.MimeType);
```

Comment:
Is it a common pattern to fall back to the json mime type by default? Should this try to use the mime type in `Microsoft.DotNet.Interactive.Formatting.Formatter.DefaultMimeType` first?

Reply:
Or should this pass `null` for the mime type here so that we use `Formatter.GetPreferredMimeTypesFor(value?.GetType())`?

Reply:
JSON is the default for value sharing. There's little use in sending text/plain or text/html except in specialized cases.